quesma/clickhouse/quesma_communicator.go

```diff
@@ -71,6 +72,18 @@ func (lm *LogManager) ProcessQuery(ctx context.Context, table *Table, query *mod
 			colName = col.ColumnName
```

Comment:
IMO, this for loop should not be part of `ProcessQuery`. In ideal case this kind of transformation should be executed before calling `ProcessQuery`

Reply:
We've now got two pipelines:
1. schema transformers
2. optimizers

Do you mean we should have the third "clean up/ compensation" pipeline?

Reply:
I would suggest something simpler, just extract this functionality to separate function that will be called before `ProcessQuery`